src/Common/CompileOptions.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.Cli
+{
+    static class CompileOptions
+    {
+        public const bool TelemetryOptOutDefault =
+#if MICROSOFT_ENABLE_TELEMETRY
+        false;
+#else
+        true;
+#endif
+    }
+}
```

Comment:
```suggestion
}

```

Make GitHub happy with an extra CR.